lib/middleware/resource.js

```diff
@@ -2,26 +2,21 @@ const { debug } = require('../log')('resource')
 
 function factory ({ loader }) {
   return async (req, res, next) => {
-    let resources = await loader.forClassOperation(req.hydra.term)
+    const classResources = await loader.forClassOperation(req.hydra.term)
+    const propertyObjectResources = await loader.forPropertyOperation(req.hydra.term)
```

Comment:
The search for property operations can be more expensive on some stores/loaders. Maybe we can add an option to skip the property operation search if there is already a class operation. Just a remark for a possible future PR. This should not be covered in this PR.